examples/install_simple/main.tf

```diff
@@ -15,29 +15,30 @@
  */
 
 module "forseti-install-simple" {
-  source  = "terraform-google-modules/forseti/google"
-  version = "~> 5.0.0"
+  source                   = "terraform-google-modules/forseti/google"
+  version                  = "~> 5.0.0"
 
-  project_id = var.project_id
-  org_id     = var.org_id
-  domain     = var.domain
+  project_id               = var.project_id
+  org_id                   = var.org_id
+  domain                   = var.domain
 
-  server_region = var.region
-  client_region = var.region
-  network       = var.network
-  subnetwork    = var.subnetwork
+  server_region            = var.region
+  client_region            = var.region
+  cloudsql_region          = var.region
```

Comment:
Only functional change is this line. The rest looks changed because I indented the values to be on the same column.